packaging/package.yml/en.md

```diff
@@ -143,32 +148,47 @@ Macro | Description
 **%python3_setup** | Runs the build portion of a setup.py using python3.
 **%python3_install** | Runs the install portion of a setup.py, to the appropriate root, using python3.
 
+### Ruby Actionable Macros
+Macro | Description
+---- | ----
+**%gem_build** | Runs gem build.
+**%gem_install** | Runs gem install with the appropriate parameters.
+
+### Waf Actionale Macros
+Macro | Description
+---- | ----
+**%waf_configure** | Runs waf configure with prefix.
+**%waf_build** | Runs waf and passes our `%JOBS%` variable.
+**%waf_install** | Runs waf install and passed the appropriate `destdir` and `%JOBS%` variable
+
 ### Qt Actionable Macros
 
 Macro | Description
 ---- | ----
 **%qmake** | Runs qmake for Qt5 with the appropriate make flags.
 **%qmake4** | Runs qmake for Qt4, as well as adding the necessary MOC, RCC, and UIC flags since those Qt4 executables end in -qt4.
 
-### Variable Macros 
+### Variable Macros
 
 Macro | Description
 ---- | ----
 **%ARCH%** | Indicates the current build architecture.
 **%CC%** | C compiler
 **%CFLAGS%** | cflags as set in `eopkg.conf`
-**%CONFOPTS%** | Flags / options for configuration, such as `--prefix=/usr`. [Full List.](https://github.com/solus-project/ypkg/blob/master/ypkg2/rc.yml#L127-L130)
+**%CONFOPTS%** | Flags / options for configuration, such as `--mandir=/usr/share/man`. [Full List.](https://github.com/solus-project/ypkg/blob/master/ypkg2/rc.yml#L217)
```

Comment:
Because I notice there is now a `%PREFIX%` variable macro.
A macro that uses a variable macro might be confusing for beginners, so I tried to find something else and mandir was the next option that did not use a variable as value.

Reply:
It doesn't matter if there is already a variable macro for it. Please revert this.